assignment_1/2_kth_element.py

```diff
@@ -0,0 +1,101 @@
+# Implement an algorithm to find the kth to last element of a singly linked list
+# Magdalena Grodzińska
+
+# linked list implementation
+
+class Node:
+    def __init__(self, data, next_node):
+        self.data = data
+        self.next_node = next_node
+
+    def get_data(self):
+        return self.data
+
+    def set_data(self):
+        self.data = new_data
+
+    def get_next(self):
+        return self.next_node
+
+    def set_next(self, new_next):
+        self.next_node = new_next
+
+
+# main function
+
+def find_kth(list, k):
```

Comment:
How many times does this iterate over the linked list? Is there a way to find the kth element with fewer iterations?

(This is a follow-up question that comes to mind here, it does not mean that your initial approach is not valid - it should work just fine)